src/module.c

```diff
@@ -1047,6 +1047,11 @@ bool Module::rootImports()
     return rootimports == 2;
 }
 
+bool Module::isCoreModule(Identifier *ident)
+{
+    return this->ident == ident && parent && parent->ident == Id::core && !parent->parent;
```

Comment:
I'm not sure, DMD does not seem to use that style, i.e. with parentheses. I get a ton of warnings about that, but not for this case :smiley: